src/test/javascript/spec/component/programming-exercise/programming-exercise.component.spec.ts

```diff
@@ -187,15 +186,6 @@ describe('ProgrammingExercise Management Component', () => {
         expect(modalService.open).toHaveBeenCalledOnce();
     });
 
-    it('should open repo export modal', () => {
-        const mockReturnValue = { componentInstance: {} } as NgbModalRef;
```

Comment:
You only deleted a test but did not add any new tests for the changes. Are they already covered sufficiently?

Reply:
The test has been moved as its directly linked to the `programming-assessment-repo-export-button.component.ts` component. (Here [src/test/javascript/spec/component/programming-assessment/programming-assessment-repo-export-button.component.spec.ts](https://github.com/ls1intum/Artemis/blob/3d99f130895d0253af001660df7dd7dea5d9e164/src/test/javascript/spec/component/programming-assessment/programming-assessment-repo-export-button.component.spec.ts))